### PR TITLE
docs: add supabase integration

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

File: content/docs/how-to/integrations/supabase.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+[Supabase](https://supabase.com/) is an open source Firebase alternative providing all the backend features you need to build a product. You can use it completely, or just the features you need.
+
+As Supabase provides full [PostgreSQL database instances](https://supabase.com/docs/guides/database), teams can use Bytebase to manage the database development lifecycle for the Supabase databases.
+
+While Supabase already has an easy-to-use GUI to conduct database operations, Bytebase adds the extra value to offer a systematic database development and change workflow. This is especially useful for cross-functional teams requiring collaboration (e.g. having dedicated DBA or platform engineering team apart from the application development teams).
+
+## Prerequisites
+
+- You need a [Supabase](https://supabase.com/) account (free signup).
+- After signup, create a supabase project, you can start with the Free plan which already includes a full PostgreSQL instance.
```
Comment (candy2255) [inline content/docs/how-to/integrations/supabase.md:17]: IMO, Bytebase workspace is a prerequisite as well.

**Review by candy2255 — APPROVED** [review]
LGTM

File: content/docs/how-to/integrations/supabase.md (hunk 1 above)
Comment (tianzhou) [reply content/docs/how-to/integrations/supabase.md:17]: I guess that's inferred.

File: content/docs/how-to/integrations/supabase.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+<hint-block type="info">
+
+If you stick with the default postgres user provided by Supabase, you will need to execute "ALTER ROLE postgres SUPERUSER" from the SQL Editor as well. See [GitHub Issue](https://github.com/supabase/supabase/issues/1856#issuecomment-1165894044).
+
+</hint-block>
+
+### Step 3 - Add the Supabase database instance to Bytebase
+
+Copy the `Host`, `Port`, `User` and `Password` from the last two steps to the form and click "Create". See [Add an Instance](/docs/get-started/configure-workspace/add-an-instance) for more details.
```
Comment (milasuperstar) [inline content/docs/how-to/integrations/supabase.md:41]:
```suggestion
Choose `PostgreSQL`, and copy the `Host`, `Port`, `User` and `Password` from the last two steps to the form and click "Create". See [Add an Instance](/docs/get-started/configure-workspace/add-an-instance) for more details.
```

File: content/docs/how-to/integrations/supabase.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+![supabase-bytebase-create-instance](/docs/how-to/integrations/supabase-bytebase-create-instance.webp)
+
+### Step 4 - Check the database instance is properly imported
+
+All databases should be sycned properly. Expect some delay if the database instance is large.
+
+![supabase-bytebase-instance](/docs/how-to/integrations/supabase-bytebase-instance.webp)
+
+So are the tables under the databases.
```
Comment (milasuperstar) [inline content/docs/how-to/integrations/supabase.md:51]:
```suggestion
So should the tables under the databases.
```

**Review by milasuperstar — COMMENTED** [review]
rest lgtm hehe

**Pull request merged**